bin/daily_total.py

```diff
@@ -63,11 +66,22 @@ def main():
             LOGGER.warning(msg)
             return
     # Format the data
-    data = "New issues on {yesterday}: {issue_count}".format(
-        yesterday=yesterday.isoformat(), issue_count=issue_count
-    )
-    # Log the data on the console
-    print(data)
+    yesterday = yesterday.isoformat()
+    count = issue_count
```

Comment:
Why do we need to rename the variable? We can reuse the same name everywhere.

Reply:
I... don't know. I'm going to say "misguided futzing" and just take that out 😁.